tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+from the Himalayas, which are included in the seismic zoning, for which a best effort higher density revisit is programmed.
+
+### Sentinel-1 orbits, modes, swaths and scenes
+
+Sentinel-1 is a **polar orbiting** platform, i.e. after crossing the North Pole, it descends to the South Pole, crosses it and then ascends back to the North Pole. This explains the *orbitProperties_pass* metadata ASCENDING (ASC) and DESCENDING (DESC) property that is tagged to each scene in the S1_GRD_FLOAT catalog. The total time needed to go from North to South Pole and back is about 98 minutes (175 orbits in 12 days). In that time, the Earth is turning about 23 degrees to the East, which is why DESC orbits are slightly rotated towards the south-west, and ASC orbits towards the north-east. The *orbitNumber_start* metadata tag list the absolute orbit sequence number (counted from start of acquisitions). The *relativeOrbitNumber_start* is often more useful, as it provides the sequence number of the orbit in the 12 day revisit cycle. Scenes from the same *relativeOrbitNumber_start* are exactly 12 days apart (6 days if you combine A and B) and have (almost) the same SAR view configuration. 
+
+Sentinel-1 can acquire in different **modes**, by programming the sensor to acquire at different resolution or polarization combinations. The most common mode over land is Interferometric Wide (IW), which is acquired in VV and VH polarization (in some maritime and polar areas in HH and HV). Extended Wide (EW) is used over maritime areas. The main trade-off between the 2 modes is between resolution and swath width, i.e. the size of the area that is illuminated by the SAR beam in range direction. The IW mode swath is approximately 250 km in size, where EW mode can cover a 400 km swath (both modes are actually composed of 3-5 subswaths).
+
+Sentinel-1 is **right looking**, i.e. the side looking antenna is sending microwave beams at a 90 degree right angle relative to the sensor's flight path (= **azimuth** direction). As stated before, a SAR collects timed range samples of the backscattered signal of a microwave pulse that is sent with a discrete frequency along the azimuth direction. The first backscattered signal is registered for the swath edge that is closest to the sensor orbit, which is called the **near range**. The last backscattered signal sample is registered at the **far range** edge of the swath. From the sensor view configuration, it is easy to figure out that near range samples have a lower **incidence angle** than far range samples, and that incidence angle varies regularly (though not linearly) over the swath. Furthermore, DESC orbits have their near to far range angles increasing from roughly east to west (rotated with the orbit angle), whereas for the ASC orbit this is roughly west to east. The combination of different incidence and **look angle** can lead to different backscattering depending on the target. 
```

Comment:
I think this is paragraph complex enough that it deserves an illustration or at least a link to an EE visualization of the angle band.

Oh, I see you talk about the angle band in the example below. Maybe mention that here?

Reply:
I have searched for a simple picture that can illustrate the various terms, but very few are useful, and I don't know about their copyright. I will most likely draw one myself. For now, I will refer to the script.